Source/multi.cpp

```diff
@@ -114,6 +114,25 @@ static BYTE *multi_recv_packet(TBuffer *pBuf, BYTE *body, DWORD *size)
 
 static void NetRecvPlrData(TPkt *pkt)
 {
+	/*
+	* We calculate _ptargx and _ptargy here and prepare for it to be sent to other clients.
+	* If px/py and ptarx/ptary don't match on other clients, then they'll move our local player to that location.
+	*/
+	{
+		// clang-format off
+		int directionOffsetX[8] = {  0,-1, 1, 0,-1, 1, 1,-1 };

```

Comment:
Would be nice if you could make your editor apply the clang-format rules.

Reply:
I turned it off there because the arrays became hard to read when you can't see the XY values side-by-side.